src/modules/emotes/style.css

```diff
@@ -98,6 +98,10 @@ img.emoji.yt-live-chat-text-message-renderer + .bttv-global-emo-5e76d399d6581c37
   margin: -0.5rem 0;
 }
 
+.bttv-emote-image-enlarged {
+  max-height: none !important;
```

Comment:
1. why is this set to important?
2. setting none as max height may actually cause undesirable behavior in some conditions. there's really no guarantee that image providers provide reasonably sized images, meaning we should always maintain an upper bound.
3. we may want to add some bottom margin to make it look a bit more organized
4. squares images may look funky, so it makes sense to radius them a tad.

```
max-height: 112px;
width: auto;
max-width: 336px;
border-radius: 0.2rem;
overflow: hidden;
```